client/contracts/bindings/consensus/consensus.go

```diff
@@ -12,12 +12,13 @@ import (
 	"github.com/kowala-tech/kcoin/client/accounts/abi/bind"
 	"github.com/kowala-tech/kcoin/client/common"
 	"github.com/kowala-tech/kcoin/client/contracts/bindings"
+	"github.com/kowala-tech/kcoin/client/contracts/bindings/kns"
 	"github.com/kowala-tech/kcoin/client/contracts/bindings/oracle"
 	"github.com/kowala-tech/kcoin/client/contracts/bindings/ownership"
 	"github.com/kowala-tech/kcoin/client/contracts/bindings/token"
 	"github.com/kowala-tech/kcoin/client/core/types"
+	kns2 "github.com/kowala-tech/kcoin/client/kns"
```

Comment:
why naming this "kns2" ?

Reply:
Because there's another package already called `kns` (`"github.com/kowala-tech/kcoin/client/contracts/bindings/kns`). There'd be a namespace conflict otherwise.